dockerfiles/docker-compose.yml

```diff
@@ -195,6 +210,21 @@ services:
       - RTD_PRODUCTION_DOMAIN
       - RTD_LOGGING_LEVEL
       - RTD_DJANGO_DEBUG
+      - RTD_SOCIALACCOUNT_PROVIDERS_GITHUB_CLIENT_ID
```

Comment:
Don't think the builders need to have access to those

Reply:
Yeah I leaned towards no too. I'll remove these and the `_key` settings.